cmd/skaffold/app/cmd/dev.go

```diff
@@ -105,6 +105,7 @@ func dev(out io.Writer, ui bool) error {
 					}
 				}
 			}
+			r.RPCServerShutdown()
```

Comment:
Also there is no Cleanup message either. That would be great to know too on the client/IDE side that the pod was deleted or not. I added #1698